build.sbt

```diff
@@ -156,7 +160,7 @@ lazy val core = (project in file("core"))
   .settings(
     commonSettings,
     functionGenerationSettings,
-    name := "glow",
+    name := s"glow_spark_${sparkVersion.value}_scala",
```

Comment:
This is a bit verbose - would this be `glow_spark_3.0.0`? Maybe we should shorten this to `glow_spark3`.

Reply:
Changed to major version only.